docs/self-managed/setup/deploy/amazon/openshift/terraform-setup-dual-region.md

```diff
@@ -25,13 +25,15 @@ If you are completely new to Terraform and the idea of IaC, read through the [Te
 
 - A [Red Hat Account](https://www.redhat.com/) to create the Red Hat OpenShift cluster.
 - An [AWS account](https://docs.aws.amazon.com/accounts/latest/reference/accounts-welcome.html) to create any resources within AWS.
-- [AWS CLI (2.17+)](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), a CLI tool for creating AWS resources.
-- [Terraform (1.9+)](https://developer.hashicorp.com/terraform/downloads)
-- [kubectl (1.30+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), a CLI tool for creating AWS resources.
+- [Terraform](https://developer.hashicorp.com/terraform/downloads)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
 - [ROSA CLI](https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/getting_started/rosa-quickstart-guide-ui#rosa-getting-started-environment-setup_rosa-quickstart-guide-ui) to interact with the cluster.
-- [jq (1.7+)](https://jqlang.github.io/jq/download/) to interact with some Terraform variables.
+- [jq](https://jqlang.github.io/jq/download/) to interact with some Terraform variables.
 - This guide uses GNU/Bash for all the shell commands listed.
 
+For the tool versions used, check the [.tool-versions](https://github.com/camunda/camunda-deployment-references/blob/feature/migrate-dual-region-rosa/.tool-versions) file in the repository. It contains an up-to-date list of versions that we also use for testing.
```

Comment:
Would this potentially cause breaking changes? Does the version change so often that it is just easier to update in the .tool-versions?

Reply:
No wouldn't cause breaking changes. We have automated tests that use the versions in that file.
This means that whoever follows this guide and looks up the relevant versions will always have a working environment.
It better reflects what we actually use and test against, rather than giving some vague (x+ version that we don't actually test for).

Later when a new version is released, a new stable branch is created, and the docs link updated to reflect the used versions for that particular Camunda release.